lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -219,6 +219,8 @@ def define
               outcome :outcome_same_sex_marriage_and_civil_partnership_not_possible
             elsif calculator.ceremony_country == "germany" && calculator.partner_is_national_of_ceremony_country?
               outcome :outcome_same_sex_civil_partnership
+            elsif calculator.ceremony_country == "romania" && calculator.partner_is_not_national_of_ceremony_country?
+              outcome :outcome_same_sex_for_none_romanians
```

Comment:
Can I actually ask you to rename it to `outcome_same_sex_marriage_in_romania_without_romanian_national`? This follows the naming convention used for the other outcomes, and particularly for `outcome_opposite_sex_marriage_in_laos_without_lao_national` (see https://github.com/alphagov/smart-answers/blob/master/lib/smart_answer_flows/marriage-abroad.rb#L167).